ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/dps-interface.yml

```diff
@@ -0,0 +1,18 @@
+### DPS interface ###
+
+dps_interface:
+  Dps1:
```

Comment:
Please change the data model to dps_interfaces as a list with max_length:1 and min_lenght:1 and a valid_values on name for Dps1 and a default value Dps1

Reply:
Fixed in latest commit - will put the model in the PR